var/spack/repos/builtin/packages/kokkos-nvcc-wrapper/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+import os
+
+
+class KokkosNvccWrapper(CMakePackage):
+    """The NVCC wrapper provides a wrapper around NVCC to make it a
+       'full' C++ compiler that accepts all flags"""
+
+    homepage = "https://github.com/kokkos/kokkos"
+    git = "https://github.com/jjwilke/kokkos-nvcc-wrapper.git"
```

Comment:
Why does this point to a developer page? Shouldn't this be part of kokkos organization?

Reply:
Yeah, I'm just going to overrule others here. The nvcc_wrapper in kokkos was deprecated, but I brought it back. Fixed.